pkg/operations/kubernetesupgrade/upgradecluster_test.go

```diff
@@ -48,7 +48,8 @@ var _ = Describe("Upgrade Kubernetes cluster tests", func() {
 		ucs := api.UpgradeContainerService{}
 		ucs.OrchestratorProfile = &api.OrchestratorProfile{}
 		ucs.OrchestratorProfile.OrchestratorType = api.Kubernetes
-		ucs.OrchestratorProfile.OrchestratorVersion = "1.6.2"
+		ucs.OrchestratorProfile.OrchestratorVersionHint = api.KubernetesVersionHint16
+		ucs.OrchestratorProfile.OrchestratorVersion = api.KubeImages[api.KubernetesVersionHint16]["version"]
```

Comment:
And that's because we will always only have one patch version of a minor version (so there will be no need to choose between one patch versus another)?